extension/src/popup/views/RecoverAccount/index.tsx

```diff
@@ -162,161 +156,117 @@ export const RecoverAccount = () => {
-                            }
-                            isTextShowing={isTextShowing}
-                            isLongPhrase={isLongPhrase}
-                            index={i}
-                          />
-                        ))}
-                      </div>
-                      {authError ? <FormError>{authError}</FormError> : <></>}
-                      <div className="RecoverAccount__mnemonic-footer">
-                        <div className="RecoverAccount__phrase-toggle">
-                          <div>{SHORT_PHRASE} word</div>
-                          <Toggle
-                            fieldSize="md"
-                            checked={isLongPhrase}
-                            id="RecoverAccount__toggle"
-                            onChange={() => setIsLongPhrase(!isLongPhrase)}
-                          />
-                          <div>{LONG_PHRASE} word</div>
-                        </div>
-                        <div className="RecoverAccount__mnemonic__text-toggle">
-                          <Button
-                            variant="secondary"
-                            onClick={() => setIsTextShowing(!isTextShowing)}
-                            size="sm"
-                            type="button"
-                          >
-                            <span> {isTextShowing ? "Hide" : "Show"}</span>
-                            {isTextShowing ? <Icon.EyeOff /> : <Icon.Eye />}
-                          </Button>
+                        <Card variant="primary">
+                          <div className="RecoverAccount__mnemonic-wrapper">
+                            <div
+                              className={`RecoverAccount__mnemonic-input ${
+                                isLongPhrase
+                                  ? "RecoverAccount__mnemonic-input--long-phrase"
+                                  : ""
+                              }`}
+                            >
+                              {phraseInputs.map((phraseInput, i) => (
+                                <PhraseInput
+                                  key={phraseInput}
+                                  phraseInput={phraseInput}
+                                  handleMnemonicInputChange={
+                                    handleMnemonicInputChange
+                                  }
+                                  isTextShowing={isTextShowing}
+                                  isLongPhrase={isLongPhrase}
+                                  index={i}
+                                />
+                              ))}
+                            </div>
+                          </div>
+                        </Card>
+                        <div className="RecoverAccount__mnemonic-footer">
+                          <div className="RecoverAccount__phrase-toggle">
+                            <div>{SHORT_PHRASE} word</div>
+                            <Toggle
```

Comment:
It looks like this element is not navigable without a mouse, tabbing through the page goes from the last word input to the show/hide button. We should make this a selection target as well.

Reply:
Ah good catch! I'll take a look